helper.fish

```diff
@@ -1653,11 +1658,18 @@ function checkLogId
   and pushd $WORKDIR/work/ArangoDB
   or begin popd; return 1; end
 
-  set -l ids (find lib arangod arangosh enterprise -name "*.cpp" -o -name "*.h" \
-    | xargs grep -h 'LOG_\(TOPIC\|TRX\|TOPIC_IF\|QUERY\|CTX\|CTX_IF\)("[^\"]*"' \
-    | grep -v 'LOG_DEVEL' \
-    | sed -e 's:^.*LOG_[^(]*("\([^\"]*\)".*:\1:')
+  if test -d client-tools
```

Comment:
same here: I wonder if the list of directories could be put into a variable so that we need to implement the grep statement just once.